frontend/taipy-gui/src/components/Taipy/Progress.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2021-2024 Avaiga Private Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+import React from "react";
+
+import Box from "@mui/material/Box";
+import CircularProgress from "@mui/material/CircularProgress";
+import LinearProgress from "@mui/material/LinearProgress";
+import Typography from "@mui/material/Typography";
+
+import { useDynamicProperty } from "../../utils/hooks";
+
+interface ProgressBarProps {
+    linear?: boolean; //by default - false
+    showValue?: boolean; //by default - false
+    value?: number; //progress value
+    defaultValue?: number; //default progress value
+}
```

Comment:
need to add a `render` dynamic boolean attribute (true by default)
ie 
```
render?: boolean;
defaultRender?: boolean;
```

Reply:
Do, I've to add these 2 attributes or replace the existing ones like - value and defaultValue with these 2?